### DISCUSSION TITLE
[Feature] Show "People" in taskbar by default

### DISCUSSION THREAD

**Discussion by Batwam** [immich-app/immich#5581] [Feature Request]
### The feature

Very basic suggestion but I was a bit surprised not to see a category for People in the sidebar of the web interface. I eventually found that this could be added if you dig through the various settings.

I would suggest to make this option `true` by default so it's easier to find and since there is room in the sidebar:
![image](https://github.com/immich-app/immich/assets/56710655/39555d7a-76cf-492b-87ff-a829fcbefe50)

I would also probably suggest doing the same with `Places` if that's not too hard to implement

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by mrjackwills** [comment]
I'm not well versed in Svelte, but I think all that needs to change is the `people: false` to `people:true` in the [preferences.store.ts](https://github.com/immich-app/immich/blob/main/web/src/lib/stores/preferences.store.ts) file.

**Reply by alextran1502** [reply]
We don't want to clutter the sidebar so we set them as false by default.

**Reply by mrjackwills** [reply]
I understand that, but it's just a single item, more importantly it's a killer feature of Immich, and also it's stored client side - so you need to update the value if you use a different browser or your session data gets deleted.
